docs/configuring.md

```diff
@@ -12,26 +12,54 @@ There are two approaches to configuring Pixeebot:
 2. **Global repository configuration:**
    Alternatively, you can create the `pixeebot.yaml` file in the `.github` directory of your `.github` repository. This will serve as a global configuration that applies to multiple repositories.
 
-## YAML
+## Properties
+
+### `ai`
 
-A typical `.yaml` configuration file might look like this:
+Contains settings related to AI functionality.
+
+#### `allow_llm_access`
+
+Setting to `false` disables Pixeebot features that [rely on generative AI](faqs.md) to analyze your code.
+
+> **Note** Pixeebot uses generative AI by default (`true` by default)
+
+Example:
 
 ```yaml
 ai:
-  allow_llm_access: true
+  allow_llm_access: false
 ```
 
-## Properties
+### `codemods`
 
-### `ai`
+Contains optional settings related to the codemod catalog Pixeebot uses to make changes to repositories.
 
-Contains settings related to AI functionality.
+#### `exclude`
 
-#### `allow_llm_access`
+A set of codemods to exclude from the catalog. Each codemod is identified by its codemod ID.
+
+Example:
+
+```yaml
+codemods:
+  exclude:
+    - pixee:python/https-connection
+```
+
+#### `prepend`
+
+A list of non-default codemods to prepend to the codemod catalog. This list is
```

Comment:
Is this intended to cover general includes, or is it a separate thing in addition to an `include` list?

Reply:
We don't have a (documented) `include` list. We have an undocumented `additional_codemods` list that prepends the list to the default catalog. This new, documented `prepend` property supplants the undocumented `additional_codemods` property and has the same behavior.

Order matters, so I don't foresee us ever adding a general `include`: it's too ambiguous about where the codemods are being inserted into the default catalog. I could see us adding an `append` to complement the `prepend`.